### DISCUSSION TITLE
IOTDB怎么实现冷热温三级数据存储？多年前的冷数据放在低廉对象存储

### DISCUSSION THREAD

**Discussion by fangzhusheng** [apache/iotdb#10036] [Q&A]
IOTDB怎么实现冷热温三级数据存储？多年前的冷数据放在低廉对象存储

**Comment by wanghui42** [comment]
企业版有这个功能https://www.timecho.com/docs/zh/UserGuide/V1.2.x/User-Manual/Tiered-Storage_timecho.html
